recipe/meta.yaml

```diff
@@ -78,57 +55,50 @@ outputs:
   - name: ray-core
     script: build-core.sh  # [not win]
     script: build-core.bat  # [win]
-    build:
-      skip: true  # [py<38]
     build:
       entry_points:
         - ray = ray.scripts.scripts:main
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - patch          # [not win]
-        - m2-patch       # [win]
         - bazel =5.4.1
-        - curl
-        - git  # [not win]
```

Comment:
You need to keep this. One of the build error is
```
  Error in fail: error running 'git init /root/.cache/bazel/_bazel_root/0d79a91d5782d685f2e1d71da6fbaf17/external/rules_perl' while working with @rules_perl:
  src/main/tools/process-wrapper-legacy.cc:80: "execvp(git, ...)": No such file or directory
```
which means `git` is missing.